src/Schema/Keywords/OneOf.php

```diff
@@ -47,23 +50,22 @@ public function validate($data, array $oneOf) : void
             Validator::each(Validator::instance(CebeSchema::class))->assert($oneOf);
 
             // Validate against all schemas
-            $matchedCount = 0;
+            $matchedCount    = 0;
+            $schemaValidator = new SchemaValidator($this->validationDataType);
```

Comment:
With stateless `SchemaValidator` we can instantiate it once

Reply:
Good for performance, indeed.